mlir/lib/Dialect/MIOpen/Transforms/AffixTuningParameters.cpp

```diff
@@ -118,6 +119,20 @@ void AffixTuningParameters::runOnOperation() {
   });
 }
 
+static void setUtilityKernelSizes(OpBuilder &b, Value arg, Operation *convOp,
+                                  Operation *funcOp) {
+  int64_t numElements = arg.getType().cast<MemRefType>().getNumElements();
+  int64_t blockSize = kUtilityKernelBlockSize;
+  int64_t elemsPerThread = kUtilityKernelElemsPerThread;
+  int64_t gridSize =
+      math_util::integer_divide_ceil(numElements, blockSize * elemsPerThread);
+  for (Operation *op : ArrayRef<Operation *>{convOp, funcOp}) {
+    op->setAttr("grid_size", b.getI32IntegerAttr(gridSize));
+    op->setAttr("block_size", b.getI32IntegerAttr(blockSize));
+    op->setAttr("elems_per_thread", b.getI32IntegerAttr(elemsPerThread));
+  }
```

Comment:
> [2022-07-14T18:51:21.027Z] 0.	Program arguments: /var/z1_miciadmin/workspace/MLIR_mlir_PR-636/build/bin/miopen-opt -miopen-affix-params -miopen-conv-to-gemm
> ...
>  [2022-07-14T18:51:21.027Z]  #4 0x00007f64c7a86200 mlir::Attribute::getDialect() const /var/z1_miciadmin/workspace/MLIR_mlir_PR-636/build/../external/llvm-project/mlir/include/mlir/IR/Attributes.h:73:0

This feels similar to previous one but in a different component, that funDialect is not loaded in `miopen-opt`